client/src/components/aboutUs.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import '../stylings/aboutUs.css';
+
+function AboutUs() {
+  return (
+    <div className='pageDiv'>
+      <h1 className='aboutUsHeader'>Reinventing the USask course catalogue. For students, by students.</h1>
+      <div className='sections'>
+      <div className='text'>
+          <p>
+          Relatable was born out of the recognition for a better and smoother way 
+          to browse for classes. What started as a class project for CMPT 370 has 
+          evolved into a powerful platform designed to enhance the educational journey 
+          for students. 
+          </p>
+          <p>
+            At Relatable, our primary goal is to simplify the process of 
+            exploring and selecting courses. We understand the frustration of navigating 
+            complex course structures and deciphering prerequisites. That's why we've 
+            developed a platform that allows students to directly see dependent courses, 
+            easily access related courses, and presents prerequisites in a clear format.
+        </p>
+        </div>
+        <img className='image1'
+          src="../images/about_us_image1.jpg"
```

Comment:
Once again, let's avoid generic names like "image1". In fact, good names for images help with SEO. Something like Relatable UofS Course Catalogue would help. Consider adding alt text to this <img> tag for a similar reason. So, we could add keywords in the alt text and make the filename more description such as Relatable Student Group.